src/geometamaker/geometamaker.py

```diff
@@ -515,6 +515,25 @@ def describe_collection(directory, depth=numpy.iinfo(numpy.int16).max,
     config = Config()
     resource = resource.replace(config.profile)
 
+    # Check if there is existing metadata for the collection
+    try:
+        existing_metadata = models.CollectionResource.load(
+            f'{directory}-metadata.yml')
+
+        # Copy any existing item descriptions from existing yml to new metadata
+        for item in resource.items:
+            # Existing metadata's item desc will always overwrite new metadata
+            # desc even if its an empty str
+            item_desc = [i.description for i in existing_metadata.items if (
+                i.path == item.path)]
+            item.description = item_desc[0] if len(item_desc) == 1 else ''
+
+        # Replace fields in existing yml if new metadata has existing value
+        resource = existing_metadata.replace(resource)
+
+    except FileNotFoundError:
+        existing_metadata = None
```

Comment:
Not that it makes any functional difference, but in this case `existing_metadata` was never defined in the first place, so we could probably just `pass` here.